one_simple_test.md

```diff
@@ -34,7 +34,7 @@ It should be red (because of rspec).
 
 ```sh
 rails g controller home index check
-rm app/views/home/check.html.erb
+rm app/views/home/check.html.slim
```

Comment:
if you follow the guide, it will be slim (i know it, since this correction is because of that) :)